pennylane/devices/experimental/default_qubit_2.py

```diff
@@ -185,7 +184,7 @@ def execute(
                     tape_resources.gate_types,
                     tape_resources.gate_sizes,
                     tape_resources.depth,
-                    Shots(execution_config.shots),
```

Comment:
Can we just do `self.tracker.update(resources=c.specs["resources"])` now?

Since there's an above comment about "temporary until shots get updated on tape".

Reply:
Good point.